opendebates/static/js/registration.js

```diff
@@ -1,14 +1,19 @@
 (function() {
   $(window).load(function() {
     $(".form-group").each(function() {
-      if ($(this).find("label").length === 1) {
-        var text = $(this).find("label").hide().text();
+      var $label = $(this).find("label");
+      if ($label.length === 1) {
+        var text = $label.text();
```

Comment:
I'm not sure I understand the reason for the changes here -- can you help me understand what's going on? `hide()` was lost on this line but I'm not sure if that was intentional or not.